### DISCUSSION TITLE
The command to clear up storage doesn't work for me

### DISCUSSION THREAD

**Discussion by balls5000** [gopro/labs#922] [Q&A] [answered]
Hello everyone,

I am trying to run 2 Dashcams in my car, one hero 11 and the other 10. The     !C16 doesn't appear to clear up 16 gigs when full, and I end up needing to manually format the microsd card.

This is the script I am using, generated with the help of QRControl.

!MWAKE=2!MQRDR=1!MDLRV=1!MSYNC=1!MTUSB=1!MDAUD=1!MBYPS=1! C16

I also used
MBOOT="!Lboot"!SAVEboot=!S!uE!O
To start recording on power and turn off when no power (when I ignite the car, the cams get USB power) 

Surely I am doing something wrong, I would have used blocky to create a more sophisticated script, but unfortunately hero 10 has no support for that.
Thanks in advance

**Comment by dnewman-gpsw** [comment, marked as answer]
There is a space in your command.  Try !C16 on its own.

HERO10 is getting support in a few weeks.  So use HERO11 and create a Blockly script.

**Comment by balls5000** [comment]
Thanks a lot, I added the   !C16 manually to the QRControl app because it wasn't available there. My SwiftKey keyboard loves pushing unnecessary spaces around. I will go and correct the script. Happy to hear that there might be a new firmware for h10. You are a saint for both replying promptly and also all the work you've done for gopro users
